appintro/src/main/res/layout-land/appintro_fragment_intro.xml

```diff
@@ -27,6 +27,7 @@
         android:layout_width="0dp"
         android:layout_height="0dp"
         android:contentDescription="@string/app_intro_image_content_description"
+        android:scaleType="centerCrop"
```

Comment:
I'm unsure about adding `centerCrop` here. What made you add this?

Reply:
I added a drive link to the image(landscape only) to this pull request comment, i'm not sure if that's what you mean.

About **centerCrop**  thing, i forgot it there. added it  when i was trying to make sure the picture size actually fill the space specified for it

Reply:
> About centerCrop thing, i forgot it there

Can I ask you to remove it?